sw/device/tests/sim_dv/spi_tx_rx_test.c

```diff
@@ -84,29 +85,29 @@ void handler_irq_external(void) {
   dif_spi_device_irq_t spi_device_irq = 0;
   switch (plic_irq_id) {
     case kTopEarlgreyPlicIrqIdSpiDeviceRxf:
-      spi_device_irq = kDifSpiDeviceIrqRxFull;
+      spi_device_irq = kDifSpiDeviceIrqRxf;
```

Comment:
Also, something else to note, these enum values are auto-generated from the IP's HJSON. Since the interrupt names in the SPI Device's HJSON are rather undescriptive, see [here](https://github.com/lowRISC/opentitan/blob/master/hw/ip/spi_device/data/spi_device.hjson#L27), they produce enum names that are less descriptive that what was manually implemented before.

To fix this, ideally, we would update `interrupt_list` fields in the IP's HJSON, but this has ramification to HW, i.e., SV files generated from this, I believe? @sriyerg

Highlighting this b/c this was **_not_** an issue for any other IP for which IRQ DIFs (and supporting enums) have been auto-generated for in #8142.

Reply:
agree. It makes sense to change `rxf` to `rx_full`. @eunchan WDYT?

Reply:
To be clear, would be nice if we could update all of them (not just `rxf`) to be more clear so that when the IRQ DIF enum gets auto-generated we are left with the same descriptive names as when this enum was manually implemented, i.e.:
(old IRQ name) --> (suggested IRQ name) --> (enum value that will get autogenerated from **suggested** IRQ name) 
`rxf` --> `rx_full` --> `kDifSpiDeviceIrqRxFull`
`rxlvl` --> `rx_above_level` --> `kDifSpiDeviceIrqRxAboveLevel`
`txlvl` --> `tx_below_level` --> `kDifSpiDeviceIrqTxBelowLevel`
`rxerr` --> `rx_error` --> `kDifSpiDeviceIrqRxError`
`rxoverflow` --> `rx_overflow`  --> `kDifSpiDeviceIrqRxOverflow`
`txunderflow` --> `tx_underflow` --> `kDifSpiDeviceIrqTxUnderflow`
`tpm_cmdaddr_notempty` --> `tpm_cmd_addr_not_empty` --> `kDifSpiDeviceIrqTpmCmdAddrNotEmpty`

Reply:
in uart, we use tx|rx_watermark. rx_above_level and tx_below_level are also fine to me. But it will be nice if we make both consistent. We can also consider to have i2c, spi_host to align with this.

Reply:
Captured this in a separate issue (#8585) and I can address in a separate PR so marking resolved.